### PR TITLE
first-frame-network walden fix

### PR DESCRIPTION
**Pull request by rbudai98**
Signed-off-by: rbudai98 <robert.budai@analog.com>

### CODE REVIEW

File: examples/first-frame-network/main.cpp [hunk 1]
```diff
@@ -61,18 +61,43 @@ int main(int argc, char *argv[]) {
...
+
     std::vector<std::string> frameTypes;
     camera->getAvailableFrameTypes(frameTypes);
     if (frameTypes.empty()) {
         std::cout << "no frame type available!";
         return 0;
     }
+
+    //set depthCompute to on or off
+    status = camera->setControl("enableDepthCompute", "on");
```
Comment (dNechita) [inline examples/first-frame-network/main.cpp:95]: enableDepthCompute is set to 'on' by default. Why was needed to be added here?

File: examples/first-frame-network/main.cpp [hunk 2]
```diff
@@ -61,18 +61,43 @@ int main(int argc, char *argv[]) {
...
+    // user can pass any config.json stored anywhere in HW
+    status = camera->setControl("initialization_config",
+                                "config/config_walden_nxp.json");
     if (status != Status::OK) {
-        LOG(ERROR) << "Could not initialize camera!";
+        LOG(ERROR) << "Could not set the initialization config file!";
         return 0;
     }
 
+    status = camera->setControl("powerUp", "call");
```
Comment (dNechita) [inline examples/first-frame-network/main.cpp:73]: "powerUp" is not used internally (we haven't come to a conclusion whether we still need to expose this control on not). So there is no reason to be called here.

Comment (rbudai98) [reply examples/first-frame-network/main.cpp]: removed

File: examples/first-frame-network/main.cpp (hunk 1 above)
Comment (rbudai98) [reply examples/first-frame-network/main.cpp]: removed

**Comment by dNechita** [comment]
These changes did make sense at the moment this PR was created but no longer make sense now. So I'm closing this PR.

**Pull request closed without merging**